deploy/assisted-installer-sso.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: assisted-installer-sso
+  namespace: REPLACE_NAMESPACE
+  labels:
+    app: bm-inventory
+type: Opaque
+stringData:
+  ocm-service.clientId: ""
```

Comment:
we can have here REPLACE_CLIENT_ID and REPLACE_CLIENT_SECRET, so it can be replaced by tools/deploy_sso_secret.py with the correct values.

Reply:
Done